.github/workflows/tests.yaml

```diff
@@ -0,0 +1,52 @@
+name: sdk-tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        description: 'MobileCoin Network'
+        required: true
+        default: 'MOBILE_DEV'
+        type: choice
+        options:
+          - 'MOBILE_DEV'
+          - 'ALPHA'
+          - 'TEST_NET'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: GCP Service Account
```

Comment:
? what is this are we running things out of GCP?

Reply:
Integration tests require the Android runtime, the CI compiles them locally and then uploads the binaries to Firebase which can run them on any device matrix needed. The credentials are needed to access Firebase.